torch/quantization/_numeric_suite_fx.py

```diff
@@ -137,3 +138,88 @@ def compare_weights_fx(float_dict, quantized_dict):
                 )
 
     return weight_dict
+
```

Comment:
Question on compare_weights_fx:  This code is identical to the eager mode implementation https://github.com/pytorch/pytorch/blob/master/torch/quantization/_numeric_suite.py#L49. For now, can we just have compare_weights_fx call compare_weights?

Reply:
Yes, they are the same .  I am calling compare_weights inside compare_weights_fx call for now.